pkg/utils/node/node.go

```diff
@@ -72,3 +72,8 @@ func GetCondition(n *v1.Node, match v1.NodeConditionType) v1.NodeCondition {
 	}
 	return v1.NodeCondition{}
 }
+
+func NotReady(n *v1.Node) bool {
+	cond := GetCondition(n, v1.NodeReady)
+	return cond.Status != v1.ConditionTrue
+}
```

Comment:
```suggestion
func Ready(n *v1.Node) bool {
	cond := GetCondition(n, v1.NodeReady)
	return cond.Status == v1.ConditionTrue
}
```

Thoughts on Ready rather than NotReady? !Ready and Ready being the true and false states makes more sense to me than NotReady and !NotReady.

Reply:
I can see a case for either. Doing `!Ready` implicitly excludes the `Unknown()` and `False()` conditions.